File: data/c#/pattern_12/code.cs
using System;
class Grammind
{
    static void Main()
    {
        int num;
        int x;
        Console.Write("Enter the number of rows and columns for the square: ");
        num = Int32.Parse(Console.ReadLine());

        for (int i = 1; i <= num; i++)
        {
            x = i;
            for (int j = 1; j <= num; j++)
            {
                Console.Write(x + " ");
                x += num;
            }
            Console.Write("\n");
        }
    }
}

File: data/java/pattern_12/code.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner createScanner = new Scanner(System.in);
        int num;
        int x;
        System.out.print("Enter the number of rows and columns for the square: ");
        num = createScanner.nextInt();

        for (int i = 1; i <= num; i++) {
            x = i;
            for (int j = 1; j <= num; j++) {
                System.out.print(x + " ");
                x += num;
            }
            System.out.print("\n");
        }
    }
}
